### PR TITLE
Adds notes about expiration output for some mc commands

### LINKED ISSUE #786: [RELEASE] mc RELEASE.2023-03-23T20-03-04Z
mc [RELEASE.2023-03-23T20-03-04Z](https://github.com/minio/mc/releases/tag/RELEASE.2023-03-23T20-03-04Z) changes that impact the docs:

- [x] [ PR #4515](https://github.com/minio/mc/pull/4515) adds expiration info to several command outputs
       - `mc admin user sts info` (this command is not yet documented, see #785)
       - `mc admin user svcacct add`
       - `mc admin user svcacct info`

### PR DESCRIPTION
**Pull request by djwfyi**
Adds info to `mc admin user svcacct` and `mc admin user sts` about exposing expiration for accounts.

Fixes some build errors from the troubleshooting doc.

Closes #786

### CODE REVIEW

File: source/reference/minio-mc-admin/mc-admin-user-sts.rst [hunk 1]
```diff
@@ -42,15 +42,15 @@ The :mc:`mc admin user sts` command has the following subcommands:
      - Description
 
    * - :mc-cmd:`mc admin user sts info`
-     - Retrieves information on the specified STS credential, including the parent user who generated the credentials and it's attached policies.
+     - Retrieves information on the specified STS credential, including the parent user who generated the credentials, attached policies, and expiration.
```
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-user-sts.rst:45]: I know `attached` is the correct term for policies, but perhaps the same `associated policies` phrasing below might work here too?

I suspect the `parent user` isn't the one attaching policies or doing something with the expiration.

File: source/reference/minio-mc-admin/mc-admin-user-svcacct.rst [hunk 2]
```diff
@@ -87,6 +87,8 @@ Syntax
                --policy "/path/to/policy.json"              \
                myminio myuser
 
+         The command returns an output of the access key and secret key for the new account.
```
Comment (feorlen) [inline source/reference/minio-mc-admin/mc-admin-user-svcacct.rst:90]: Is `an output of` significant here, or could this be `returns the access key...` instead?

**Review by feorlen — CHANGES_REQUESTED** [review]
Noticed a typo, also a couple ideas you might like.

File: source/reference/minio-mc-admin/mc-admin-user-sts.rst (hunk 1 above)
Comment (djwfyi) [reply source/reference/minio-mc-admin/mc-admin-user-sts.rst:45]: Matching what we say below makes sense!

File: source/reference/minio-mc-admin/mc-admin-user-svcacct.rst (hunk 2 above)
Comment (djwfyi) [reply source/reference/minio-mc-admin/mc-admin-user-svcacct.rst:90]: Not significant, cut.

File: source/reference/minio-mc-admin/mc-admin-user-svcacct.rst [hunk 3]
```diff
@@ -224,7 +226,15 @@ Syntax
 .. mc-cmd:: info
    :fullpath:
 
-   Returns a description of a access keys associated to the specified user. The description includes the parent user of the specified access keys, its status, and whether the access keys has an assigned inline policy.
+   Returns a description of a specified access keys.
```
[the remark this answers is not in the document]
Comment (djwfyi) [reply source/reference/minio-mc-admin/mc-admin-user-svcacct.rst:229]: Indeed. Also switched to the definite article.

**Review by feorlen — APPROVED** [review]
👍🏻

**Pull request merged**